SectionGrid.js

```diff
@@ -128,7 +133,12 @@ const SectionGrid = memo(
 
     const groupSectionsFunc = useCallback(
       (section) => {
-        const chunkedData = chunkArray(section.data, itemsPerRow);
+        let chunkedData = chunkArray(section.data, itemsPerRow, true);
```

Comment:
I am not sure why we need the third parameter (`true`) here.

Reply:
It is a useless parameter. I will remove it.